rmw_fastrtps_shared_cpp/src/rmw_node.cpp

```diff
@@ -246,11 +265,32 @@ __rmw_create_node(
   // since the participant name is not part of the DDS spec
   participantAttrs.rtps.setName(name);
 
+  bool leave_middleware_default_qos = false;
+  const char * env_var = "RMW_FASTRTPS_USE_QOS_FROM_XML";
+  // Check if the configuration from XML has been enabled from
+  // the RMW_FASTRTPS_USE_QOS_FROM_XML env variable.
+  char * config_env_val = nullptr;
+#ifndef _WIN32
```

Comment:
Same as above, this functionality is in `rcutils.`

Reply:
Please, check the comment in the previous conversation.